### PR TITLE
LPS-169866: Fix SXPBlueprintLocalServiceTest and SXPElementLocalServiceTest updating elemnts with the same ERC

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Creating new test case, expecting to throw a `Duplicate<EntityName>ExternalReferenceCodeException.`

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-169866

### CODE REVIEW

**Review by peerkar — COMMENTED** [review]
Hi @sergiojimcos added a couple minor notes to check. Not sure if there was a specific reason to test updating twice with the same signature so that change can be ignored if unsure. Thanks!

File: modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/service/test/SXPElementLocalServiceTest.java [hunk 1]
```diff
@@ -181,6 +156,19 @@ public void testUpdateSXPElement() throws Exception {
 		Assert.assertEquals("1.2", sxpElement1.getVersion());
```
[the remark this answers is not in the document]
Comment (sergiojimcos) [reply modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/service/test/SXPElementLocalServiceTest.java:156]: make sense @peerkar!

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:relevant

**Review by peerkar — APPROVED** [review]

**Comment by peerkar** [comment]
ci:forward

**Pull request closed without merging**